app/components/input/index.ts

```diff
@@ -1,3 +1,3 @@
-export { InputSearch } from './input-search'
 export { InputForm } from './input-form'
+export { InputIcon } from './input-icon'
```

Comment:
Is it ok to use relative path with dot ?
ref: https://github.com/tabi-memo/frontend/pull/7#discussion_r1406898337

But for me, it's ok to use especially when we import a file from the same directory.

Reply:
Yes... I realized that it is okay (or even better) to use relative path if it's on the same directory so I actually changed since I realized it. Sorry 🙏